packages/builder/src/definition.ts

```diff
@@ -83,7 +83,12 @@ export class ChainDefinition {
         }
 
         const fullActionName = `${action}.${name}`;
-        actionNames.push(name);
+
```

Comment:
@dbeal-eth Im a little curious about this change and would appreciate your input.

Before on one of the snx builds we were getting a duplicate error due to a setting taking up the name, now that setting is a step I figure we dont want to do this but this was the only way I could restore backwards compatibility without breaking the rest of the 'var' action registering.

Reply:
Not sure what your asking here, need to look closer over weekend perhaps

Reply:
Not sure how they can have duplicate names. Could you share example?

Reply:
In synthetix-sandbox we have this setting here:

https://github.com/Synthetixio/synthetix-sandbox/blob/main/cannonfile.toml#L11

and then there is a provision step by the same name:

https://github.com/Synthetixio/synthetix-sandbox/blob/main/cannonfile.toml#L37